### ISSUE TITLE
TFLite Converter, add possibility to ignore some OPs from quantization

### ISSUE THREAD

**Issue by gaikwadrahul8** [google-ai-edge/ai-edge-torch#387]
### Issue type

Feature Request

### Have you reproduced the bug with TensorFlow Nightly?

No

### Source

binary

### TensorFlow version

v2.13.0-17-gf841394b1b7

### Custom code

No

### Python version

3.10.13

### Current behavior?

Quantizing models to integer works as expected, but because some of the final operations work in INT8, a large accuracy drop can be observed in some models.

This ticket is a feature request to be able to exclude specific operations from quantization and execute in FP32. OpenVINO supports this feature as `ignored_scope` param during quantization. [Link to OpenVINO quantizer documentation.](https://docs.openvino.ai/2022.3/basic_qauntization_flow.html#tune-quantization-parameters) Considering how Edge TPU works, the solution should be to set where to stop quantization and execute the rest of the OPs in FP32 on the CPU.

Lets take yolov8n as an example and convert the pytorch model to TF using onnx2tf. Lets compare the main branch in [FULL INT8](https://github.com/adamp87/ultralytics/tree/main) quantization, with a dirty hack by detaching the last operations and executing as [INT8 + FP32](https://github.com/adamp87/ultralytics/tree/tflite_detach_dirty). As a note, Edge TPU compiled models larger than 192pixel input execute the head on the CPU as some Transpose operations are too large for the TPU.

| Model yolo8n  | mAP50 | mAP50-95 | Note | Speed on Intel CPU |
| ------------- | ------------- | ------------- | ------------- | ------------- |
| Baseline FP32 |  52.6 | 37.4  | Main branch | N/A |
| TFLite Full INT8  |  48.8 | 32.9 | per-tensor | 162.2 ms |
| TFLite INT8 + FP32 |  50.3 | 35.2 | per-tensor | 166.0ms |
| TFLite Full INT8  |  49.8 | 33.9 | per-channel | N/A |
| TFLite INT8 + FP32  |  51.4 | 36.3 | per-channel | N/A |

### Standalone code to reproduce the issue

```shell
https://github.com/adamp87/ultralytics/blob/tflite_detach_dirty/yolo8_full_int8_nohead_test.ipynb
```

**Comment by gaikwadrahul8** [comment]
This issue originally reported by @adamp87 has been moved to this dedicated repository for [ai-edge-torch](https://github.com/google-ai-edge/ai-edge-torch) to enhance issue tracking and prioritization. To ensure continuity, we have created this new issue on your behalf.

We appreciate your understanding and look forward to your continued involvement.

**Comment by shivvaam0001** [comment]
Hello,

Thank you for sharing this feature request with us! This functionality has been implemented in our [latest release](https://github.com/google-ai-edge/litert-torch/releases/tag/v0.9.1). We are closing this request now. 

Thank you again for your valuable feedback and continued support!
